packages/dev/core/src/AudioV2/abstractAudio/subNodes/abstractAudioSubGraph.ts

```diff
@@ -59,21 +93,44 @@ export abstract class _AbstractAudioSubGraph {
 
     /**
      * Gets a previously created sub node.
+     *
      * @param name - The name of the sub node
      * @returns The named sub node, or `null` if it has not been created, yet
+     *
      * @internal
      * */
-    public getSubNode<T extends AbstractNamedAudioNode>(name: string): Nullable<T> {
+    public getSubNode<T extends _AbstractAudioSubNode>(name: string): Nullable<T> {
         return (this._subNodes[name] as T) ?? null;
     }
 
+    /**
+     * Removes a sub node from the sub graph.
+     *
+     * @param subNode - The sub node to remove
+     * @returns A promise that resolves when the sub node is removed
+     *
+     * @internal
+     */
+    public async removeSubNode(subNode: _AbstractAudioSubNode): Promise<void> {
```

Comment:
```suggestion
    public async _removeSubNodeAsync(subNode: _AbstractAudioSubNode): Promise<void> {
```

Reply:
As per the similar suggestion above, I'm going to leave this as-is for this PR because there a bunch more of these functions that should follow the same convention and I'll do them all in a separate PR.